src/DataCollectors/TraceDataCollector/VanguardCollector/DynamicCoverageDataCollector.cs

```diff
@@ -10,8 +10,10 @@ namespace Microsoft.VisualStudio.Coverage
     using System.Xml;
     using Interfaces;
     using Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection;
+    using Microsoft.VisualStudio.TestPlatform.PlatformAbstractions;
```

Comment:
I believe platform abstraction is core part of TP, & any extension should not take a dependency on it.

Reply:
Added code in TDC for getting current OS name.